express/parsers/structure.py

```diff
@@ -41,6 +68,86 @@ def __init__(self, *args, **kwargs):
         self.lattice_only_structure = mg.Structure.from_str(self.structure_string, self.structure_format)  # deepcopy
         self.lattice_only_structure.remove_sites(range(1, len(self.structure.sites)))
 
+    def get_inchi_run(self):
+        return inchi_run
+
+    def get_inchi_null(self):
+        """
+        Function returns "Not Available" value for InChI when it cannot be calculated.
+
+        Returns:
+            Str
+        """
+        inchi_key_str = {
+            'name': 'inchi',
+            'value': ''
+        }
+        return inchi_key_str
+
+    def create_pybel_smi_from_poscar(self):
+        """
+        Function using ase to convert the POSCAR formatted string of a structure
+        into an XYZ formatted text file for that structure.
+
+        Then pybel converts the XYZ formatted text file into a SMILES format.
+
+        Returns:
+            Str
+        """
+        fi = "geom.xyz"
```

Comment:
fi, fstr, ase_pos, psmi - are all very difficult to understand. Let's use full names, please, here and below.